### PR TITLE
fix note sharing broken issue

### PR DESCRIPTION
**Pull request by ronplaymeta**
Hi @pablo-mayrgundter.
I fixed Note sharing breaking issue when user selects a note and reload page, please check the result.
Thank you.

### CODE REVIEW

File: src/Components/SideDrawer/SideDrawer.jsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export default function SideDrawer() {
     if (noteHash !== undefined) {
       const extractedCommentId = noteHash.split(':')[1]
       setSelectedNoteId(Number(extractedCommentId))
+      console.log('isDrawerOpen: ', isDrawerOpen)
```
Comment (OlegMoshkovich) [inline src/Components/SideDrawer/SideDrawer.jsx:46]: remove.

Comment (ronplaymeta) [reply src/Components/SideDrawer/SideDrawer.jsx]: @OlegMoshkovich, thank you, i removed all `console.log`.

File: src/WidgetApi/Utils.js [hunk 2]
```diff
@@ -21,7 +21,6 @@ export default class Utils {
    */
   getSelectedElementIds(state) {
     const elementIds = []
-    // console.log(state.selectedElements)
```
Comment (pablo-mayrgundter) [inline src/WidgetApi/Utils.js:24]: Remove

**Review by pablo-mayrgundter — APPROVED** [review]
Heya, just one debug statement to remove.

Thanks!

**Comment by pablo-mayrgundter** [comment]
Fixes #575

**Pull request merged**